### ISSUE TITLE
[Bug] Example basic voice broken? 

### ISSUE THREAD

**Issue by timmermansjoy** [HuyaneMatsu/hata#89]
### Description

When trying to implement the example [basic voice](https://github.com/HuyaneMatsu/hata/blob/master/docs/examples/e11_basic_voice/main.py) I cant even let the bot join because `AttributeError: 'InteractionEvent' has no attribute '.voice_state'.` 

### Reproduction Steps

run the example code

### Reproduction Code

```python
@Sakuya.interactions(guild = MY_GUILD)
async def join(client, event):
    """Joins to voice channel."""
    # Getting the author voice state
    voice_state = event.voice_state
    if voice_state is None:
        return 'You are not at a voice channel!'
    
    # Connecting the client to the same channel, where the user is.
    try:
        await client.join_voice(voice_state.channel)
    except TimeoutError:
        # Could not connect.
        return 'Timed out meanwhile tried to connect.'
    
    except RuntimeError:
        # Not every required library is installed.
        return 'The client cannot play voice, some libraries are not loaded.'
    
    # Great success.
    return f'Joined to {voice_state.channel.name}'
```

### Expected behaviour

the bot to join

### Actual behaviour

AttributeError thrown

### System information

Hata version: 1.3.53
Scarletio version: 1.0.68
Platform: macOS-14.2.1-arm64-arm-64bit
Python implementation: cpython
Python version: 3.11 final

**Comment by HuyaneMatsu** [comment]
`event.voice_state` should never return the user's voice state, so it is not a missing feature, but the example is bad. I will update the example next update, until please replace:
```py
    # Getting the author voice state
    voice_state = event.voice_state
    if voice_state is None:
        return 'You are not at a voice channel!'
```
with
```py
    # Getting the invoking user's voice state
    voice_state = event.guild.voice_states.get(event.user_id, None)
    if voice_state is None:
        return 'You are not at a voice channel!'
```

**Comment by timmermansjoy** [comment]
after this I still get


```
  File "/Users/joy/fun/discord/.venv/lib/python3.11/site-packages/hata/ext/slash/command/slash_command/slash_command_function.py", around line 168, in invoke
    await process_command_coroutine(
        client,
        interaction_event,
        self.response_modifier,
        command_coroutine,
    )
  File "/Users/joy/fun/discord/.venv/lib/python3.11/site-packages/hata/ext/slash/responding.py", line 376, in process_command_coroutine
    response = await coroutine
  File "/Users/joy/fun/discord/penny/main.py", line 42, in join
    await client.join_voice(voice_state.channel)
  File "/Users/joy/fun/discord/.venv/lib/python3.11/site-packages/hata/discord/client/compounds/client_gateway.py", line 238, in join_voice
    voice_client = VoiceClient(self, guild_id, channel_id)
  File "/Users/joy/fun/discord/.venv/lib/python3.11/site-packages/hata/discord/voice/voice_client.py", line 185, in __new__
    self._encoder = OpusEncoder()
  File "/Users/joy/fun/discord/.venv/lib/python3.11/site-packages/hata/discord/voice/opus.py", line 243, in __new__
    _set_default_encoder_settings(encoder)
  File "/Users/joy/fun/discord/.venv/lib/python3.11/site-packages/hata/discord/voice/opus.py", line 198, in _set_default_encoder_settings
    opus.opus_encoder_control(encoder, SET_BITRATE, 131072)
  File "/Users/joy/fun/discord/.venv/lib/python3.11/site-packages/hata/discord/voice/opus.py", line 112, in _err_lt
    raise OpusError(result)
OpusError('invalid argument')
```

**Comment by HuyaneMatsu** [comment]
I cant reproduce that issue on linux, so I cannot fix it.

I can only recommend for you to check your opus installation.

**Comment by timmermansjoy** [comment]
I'm on a Mac and installed [opus from brew](https://formulae.brew.sh/formula/opus) Do I need to install something else? there is nothing mentioned about opus in the readme?

**Comment by HuyaneMatsu** [comment]
i could add to readme that you should install Asahi linux on mac.

**Comment by timmermansjoy** [comment]
I mean I'm willing to help but not gonna waste my time if you gonna start trolling? 

What version of opus are you running? 

```
opusenc -V
opusenc opus-tools 0.2 (using libopus 1.5.1)
Copyright (C) 2008-2018 Xiph.Org Foundation
```

I tried Pycord and it looks like they working with the [same library](https://docs.pycord.dev/en/stable/_modules/discord/opus.html) and I got it working.

This library does look nicer to work with and think there is just something little wrong / miss configured
